book/src/integrate/deps.md

```diff
@@ -65,3 +65,20 @@ Add these lines to `Cargo.toml`:
 +flutter_rust_bridge = "1"
 ```
 
+## System dependencies
+
+For non-debian based Linux distributions, there are a few prerequisites:
+Ensure that packages are up to date (or install by demand).
+
+- clang
+- llvm-libs
+- glibc
+
+> restart may be required
```

Comment:
I moved parts, so now it is more precise.
How to better style this part?


Please check when you'll have time

Reply:
That LGTM, I will restyle after merge. Good job!

Reply:
Ok! Thank you!